policy/modules/system/logging.te

```diff
@@ -234,6 +234,7 @@ optional_policy(`
 	init_list_unit_dirs(auditd_t)
 	systemd_start_power_units(auditd_t)
 	systemd_status_power_units(auditd_t)
+	systemd_stream_connect_userdb(auditd_t)
```

Comment:
probably should just call `auth_use_nsswitch()` its resolving names which implies it reads /etc/passwd and /etc/nsswitch